scripts/docker-compose.libc.yml

```diff
@@ -0,0 +1,280 @@
+# require {docker-compose.core.yml, docker-compose.dev.yml}
```

Comment:
@come2ry 
Is it intentional that only `libc` is listed in a separate file?

Reply:
I made a small correction in #128.
If this is intentional and you do not want the fix, please comment.

Reply:
@mdonaka 
It was a pure mistake.
Thank you very much.